src/p5xr/core/p5xrInput.js

```diff
@@ -63,6 +64,18 @@ export default class p5xrInput {
     const p = this._pose?.transform?.position;
     return new p5.Vector(p.x, p.y, p.z);
   }
+  /** @returns {p5.Vector} Returns the current rotation as a Vector */
```

Comment:
A good idea to mention the gimbal locking behavior here and provide `applyMatrix(p5xrInput.pose)` as an alternative.